src/TabCompletion/Matcher/AbstractContextAwareMatcher.php

```diff
@@ -56,10 +56,24 @@ protected function getVariable($var)
     /**
      * Get all variables in the current Context.
      *
+     * @param bool $dollarPrefix
+     *   Whether to prefix '$' to each variable name.
+     *
      * @return array
      */
-    protected function getVariables()
+    protected function getVariables($dollarPrefix = false)
```

Comment:
I'm not sure whether anything else will ever use it; but if they do, I think it would be nicer if it was as simple as passing an argument to the existing function, so I'd be inclined to leave it where it is.